HTTP/src/HTTPClient/HttpURLConnectionClient.java

```diff
@@ -19,6 +22,7 @@ public static void main(String[] args) throws Exception {
         HttpURLConnectionClient http = new HttpURLConnectionClient();
         System.out.println("Sent Http GET request");
         http.sendGet();
```

Comment:
This sends an extra request. It's a leftover from the example text and can be removed. The previous line should also be removed as it's just logging for this request. 

This is why you get the following in output:

```
Sent Http GET request
http://localhost:8080java+sdk
Response Code: 200
Request Headers
Set-cookie : [userID=Cookie Monster]


response: </br></br></br>Sun Feb 18, 2018 2:39 PM</br></br>asdfsadfasdfsadfasdf</br></br>
```